worlds/oot/Patches.py

```diff
@@ -2110,22 +2111,19 @@ def place_shop_items(rom, world, shop_items, messages, locations, init_shop_id=F
         else:
             if location.item.trap:
                 item_display = location.item.looks_like_item
```

Comment:
hm, I did not realize before but the fact that I can't put new attributes on some items means that this can't work. previously, all trap items in OoT were given false models to disguise them; I can't store that false model on the item now, I have to generate it here. this is sort of undesirable but I no longer have a choice.